packages/ListBox/stories/examples/multi.js

```diff
@@ -1,257 +0,0 @@
-    {characters.antiHeroes}
-    <ListBox.Option isSelected>Mad Max</ListBox.Option>
-    <ListBox.Divider>Villians</ListBox.Divider>
-    {characters.villians}
-    <ListBox.Divider>Heroes</ListBox.Divider>
-    <ListBox.Option isSelected>Aquaman</ListBox.Option>
-    {characters.heroes}
-  </ListBox>
-);
-
-const styles = {
-  container: css`
-    padding: 12px;
-  `,
-  button: css`
-    font-size: 16px;
-    margin: 2px;
-    padding: 8px;
-    ${({ isSelected }) => {
-      return isSelected ? `background: ${tokens.color.blue}; color: ${tokens.color.white}` : "";
-    }}
-  `,
-};
-
-export const ControlledIsSelected = () => {
-  const [options, setOptions] = React.useState(characters.antiHeroesRaw);
-
-  const handleClick = index => () => {
-    setOptions(() => {
-      const cloneOptions = options.slice(0);
-
-      cloneOptions[index].isSelected = !cloneOptions[index].isSelected;
-
-      return cloneOptions;
-    });
-  };
-
-  function handleChange(selectedOptions) {
-    /** this will be trigger when a uncontrolled change occurred inside the list-box
-    let's sync our local state with the one on the list-box so we can turn on / off
-    the buttons */
-
-    setOptions(options => {
-      const cloneOptions = options.slice(0);
-      cloneOptions.forEach((op, index) => {
-        cloneOptions[index].isSelected = selectedOptions.includes(index);
-      });
-
-      return cloneOptions;
-    });
-  }
-
-  return (
-    <React.Fragment>
-      <div css={styles.container}>
-        <p>Click on any button to controlled the ListBox:</p>
-        {options.map((item, index) => (
-          <button
-            css={styles.button}
-            type="button"
-            key={item.label}
-            isSelected={options[index].isSelected}
-            onClick={handleClick(index)}
-            data-pka-anchor={`button_${index}`}
-          >
-            {item.label}
-          </button>
-        ))}
-      </div>
-      <hr />
-      <ListBox isInline isMulti onChange={handleChange}>
-        {options.map(item => {
-          return (
-            <ListBox.Option key={item.label} isSelected={item.isSelected}>
-              {item.label}
-            </ListBox.Option>
-          );
-        })}
-      </ListBox>
-    </React.Fragment>
-  );
-};
-
-export const DefaultIsSelected = () => {
-  return (
-    <ListBox
-      isInline
-      isMulti
-      onChange={selectedOptions => {
-        console.log(selectedOptions);
-      }}
-    >
-      {characters.antiHeroesRaw.map((item, index) => {
-        return (
-          <ListBox.Option key={item.label} defaultIsSelected={Boolean(index % 2)}>
-            {item.label}
-          </ListBox.Option>
-        );
-      })}
-    </ListBox>
-  );
-};
-
-export const TriggerIsHidden = () => {
-  return (
-    <ListBox
-      isInline
-      isMulti
-      onChange={selectedOptions => {
-        console.log(selectedOptions);
-      }}
-    >
-      <ListBox.Trigger isHidden />
```

Comment:
We should make sure this little nugget gets kept somewhere.  Maybe just reinstated in a later PR, but let's make sure we don't forget.